### ISSUE TITLE
代码问题

### ISSUE THREAD

**Issue by TccccD** [lianqing11/PyCDA#5]
images, fake_labels, _ = batch
...
fake_mask = (fake_labels!=255).float().detach()
conf_mask = torch.gt(conf_tea, args.conf_threshold).float().detach()
loss_pseudo = loss_pseudo * conf_mask.detach() * fake_mask.detach()

这里fake_labels 是target的gt 还是pseudo label，如果是pseudo label的话是从哪个模型推理得到的，source only吗？但如果是gt的话，target的gt就参与了训练了啊。

**Comment by TccccD** [comment]
哦.....我发现
label = np.zeros((560, 480), dtype=np.uint8)
label = Image.fromarray(label)
这里的label是np.zeros....
